tests/test_ekubomath.py

```diff
@@ -1,4 +1,126 @@
+
+testdata_get_liquidity = [
+    (
+        0, #x
+        2147718844699912146 * bool_to_sign(1),#y
+        math.sqrt(1.000001 ** (158482 * bool_to_sign(0))), #sp
+        math.sqrt(1.000001 ** (158400 * bool_to_sign(0))), #sa
+        math.sqrt(1.000001 ** (158440 * bool_to_sign(0))), #sb
+        99209107691145427163307 * bool_to_sign(1) # L
+    ),
+    (
+        2380000000000000000 * bool_to_sign(0),
+        3064071012072246010 * bool_to_sign(0),
+        math.sqrt(1.000001 ** (158474 * bool_to_sign(0))),
+        math.sqrt(1.000001 ** (158240 * bool_to_sign(0))),
+        math.sqrt(1.000001 ** (158860 * bool_to_sign(0))),
+        17442970835766897709847 * bool_to_sign(0)
+
+    ),
+    (
+        179901791914979877 * bool_to_sign(1),
+        0,
+        math.sqrt(1.000001 ** (158574 * bool_to_sign(0))),
+        math.sqrt(1.000001 ** (158580 * bool_to_sign(0))),
+        math.sqrt(1.000001 ** (158840 * bool_to_sign(0))),
+        1498166625277028332296 * bool_to_sign(1)        
+    )   
+]
+
+testdata_get_x = [
+    (          
+        99209107691145427163307 * bool_to_sign(1), #L
+        math.sqrt(1.000001 ** (158482 * bool_to_sign(0))), # sp
+        math.sqrt(1.000001 ** (158400 * bool_to_sign(0))), # sa
+        math.sqrt(1.000001 ** (158440 * bool_to_sign(0))), # sb
+        0  # Expected result is 0 #x
+    ),
+    (
+        17442970835766897709847 * bool_to_sign(0), #L
+        math.sqrt(1.000001 ** (158474 * bool_to_sign(0))), # sp
+        math.sqrt(1.000001 ** (158240 * bool_to_sign(0))), # sa
+        math.sqrt(1.000001 ** (158860 * bool_to_sign(0))), # sb
+        2380000000000000000 * bool_to_sign(0),
+    ),
+    (
+        1498166625277028332296 * bool_to_sign(1), #L
+        math.sqrt(1.000001 ** (158574 * bool_to_sign(0))), # sp
+        math.sqrt(1.000001 ** (158580 * bool_to_sign(0))), # sa
+        math.sqrt(1.000001 ** (158840 * bool_to_sign(0))), # sb
+        179901791914979877 * bool_to_sign(1) #x
+    )
+]
+
+
+testdata_get_y = [
+    (          
+        99209107691145427163307 * bool_to_sign(1), #L
+        math.sqrt(1.000001 ** (158482 * bool_to_sign(0))), # sp
+        math.sqrt(1.000001 ** (158400 * bool_to_sign(0))), # sa
+        math.sqrt(1.000001 ** (158440 * bool_to_sign(0))), # sb
+        2147718844699912146 * bool_to_sign(1)  # Expected result is 0 #y
+    ),
+    (
+        17442970835766897709847 * bool_to_sign(0), #L
+        math.sqrt(1.000001 ** (158474 * bool_to_sign(0))), # sp
+        math.sqrt(1.000001 ** (158240 * bool_to_sign(0))), # sa
+        math.sqrt(1.000001 ** (158860 * bool_to_sign(0))), # sb
+        3064071012072246010 * bool_to_sign(0)  # Expected result is 0 #x 
+    ),
+    (
+        1498166625277028332296 * bool_to_sign(1), #L
+        math.sqrt(1.000001 ** (158574 * bool_to_sign(0))), # sp
+        math.sqrt(1.000001 ** (158580 * bool_to_sign(0))), # sa
+        math.sqrt(1.000001 ** (158840 * bool_to_sign(0))), # sb
+        0  # Expected result is 0 #y
+    )
+]
```

Comment:
No tests written for get_capital_token1_terms_from_L, get_capital_in_token1_terms_fromy, get_liquidity_from_token1_capital, impermanent_loss_absolute